### DISCUSSION TITLE
Can't Connect to Webmail after fresh install

### DISCUSSION THREAD

**Discussion by kargenti** [modoboa/modoboa#2472] [Q&A]
Hey fellas. 

I'd like to preface this with I am a linux newbie who is being tasked with way too much given my experience-to-workload ratio. But my boss doesn't think I'll learn any other way.

I'll just run through my process real quick.

1. Host a Debian 11 VM with a bridged connection to connect it to our network.
2. Follow the easy install instructions direct from modoboa.
3. Profit.

Except, at #2 I am running into an issue. 

The installation goes fine. I'm able to see the apache server is running. But when I enter my test url, even after changing my hosts files, I get connection errors.

127.0.0.1 brings up the Apache2 Debian Default page.
mail.testmodoboa.local brings up "This site can't be reached."

I feel like the answers are obvious, but I'm following the official guide, and trying dozens of others, and it just never seems to work out.

Please take it easy on me, this is like, only one of 10 insane projects dumped on me this year.

**Comment by tonioo** [comment]
@kargenti You followed the manual installation instructions? You did not use the installer?
